src/onegov/feriennet/models/notification_template.py

```diff
@@ -135,19 +118,15 @@ def bind(
         self.bound[token] = method
 
     def render(self, text: str) -> Markup:
-        text = escape(text)
```

Comment:
This still needs to happen, we also use `render` for the subject. You will need to change the notification template views to wrap `self.text` in `Markup`.

Ideally we'd add a new database type that automatically creates `Markup` instead of `str` for fields that contain `Markup`. But we can do that in a separate step, we would also have to figure out how to get the same functionally in `data_property`.